### ISSUE TITLE
Allow license to be spelled as "licence"

### ISSUE THREAD

**Issue by mlntr** [microsoft/vscode-vsce#870]
```
WARNING  LICENSE.md, LICENSE.txt or LICENSE not found
Do you want to continue? [y/N]
```
It doesn't find `licence` with `c`, which is very common/correct (GitHub accepts it for instance)

**Comment by joaomoreno** [comment]
PR welcome!

**Comment by DonJayamanne** [comment]
@joaomoreno Is there a way to install the pre-release bits, or just verify the fact that the changes are in main?

**Comment by roblourens** [comment]
Verified by code review

**Comment by joaomoreno** [comment]
Sorry about that, just published v2.20.0.
